demo-shell/src/app/components/files/files.component.html

```diff
@@ -362,7 +362,8 @@
                         <adf-version-manager
                             [node]="documentList.selection[0].entry"
                             [showComments]="showVersionComments"
```

Comment:
Why one property is enable and another allow as prefix? Can we define a standard?

Reply:
What we use more and what you suggest? "enable" or "allow"? :) I am fine with anything

Reply:
changed to allow, as there are more "allow" usages for this feature and only 1 "enable"

Reply:
Good , I think we need to have a bigger discussion around all the property prefix for 3.0